src/modules/select/wallets/ownbit.ts

```diff
@@ -0,0 +1,40 @@
+import { mobileWalletInstallMessage } from '../content'
+import {
+  WalletModule,
+  Helpers,
+  InjectedWithBalanceOptions
```

Comment:
@jtomtan `InjectedWithBalanceOptions` is an unused import, please remove 👍

Reply:
nice catch! 👍 InjectedWithBalanceOptions removed.